Fix to clarify that an X.509 SVID must contain one URI SAN

amartinezfayo opened the pull request:
Change  from "An X.509 SVID MUST NOT contain more than one URI SAN" to "An X.509 SVID MUST contain one URI SAN"

amartinezfayo committed:
Fix to clarify that an X.509 SVID must contain one URI SAN

Change  from "An X.509 SVID MUST NOT contain more than one URI SAN" to "An X.509 SVID MUST contain one URI SAN"

evan2645 commented on standards/X509-SVID.md:
```diff
@@ -33,7 +33,7 @@ Perhaps the most important function of SPIFFE is to secure process to process co
 
 ## 2. SPIFFE ID
-In an X.509 SVID, the corresponding SPIFFE ID is set as a URI type in the Subject Alternative Name extension (SAN extension, see [RFC 5280 section 4.2.16][2]). An X.509 SVID MUST NOT contain more than one URI SAN. It MAY contain any number of other SAN fields, including DNS SANs.
+In an X.509 SVID, the corresponding SPIFFE ID is set as a URI type in the Subject Alternative Name extension (SAN extension, see [RFC 5280 section 4.2.16][2]). An X.509 SVID MUST contain one URI SAN. It MAY contain any number of other SAN fields, including DNS SANs.
```
Perhaps we can change this to `MUST contain exactly one` or `MUST contain only one`? In the proposed wording, I feel that the "one and only one" semantics are not clear

amartinezfayo replied:
I agree. I think that `MUST contain exactly one` is more clear. I'll change to that.

amartinezfayo committed:
Change to say "MUST contain exactly one URI SAN"

Change in the text to say "MUST contain exactly one URI SAN" based on feedback.

evan2645 reviewed (approved):
LGTM

This change needs at least one additional approver

dudedad commented:
LGTM.  Agreed.
